german/abwarten/abwarten.md

```diff
@@ -0,0 +1,6 @@
+In der Tat lösen sich so einige Probleme von selbst, wenn man einfach nur lange genug wartet.
+
+Bei genauerer Betrachtung ist die aktuelle allerdings keine solche Situation, in der die sich ergebende Lösung besonders angenehm
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.